containers-tests/tests/Utils/Strictness.hs

```diff
@@ -27,6 +27,12 @@ instance Arbitrary a => Arbitrary (Bot a) where
     , (4, Bot <$> arbitrary)
     ]
 
+instance CoArbitrary a => CoArbitrary (Bot a) where
+  coarbitrary (Bot x) = coarbitrary x
+
+instance Function a => Function (Bot a) where
+  function = functionMap (\(Bot x) -> x) Bot
+
```

Comment:
These shouldn't be necessary.
In fact if we need them, we're doing something odd. There is no reason to generate functions that take in `Bot a`.

Reply:
I don't know why you say that. It avoids having to unwrap the `Bot`s manually.

Reply:
> It avoids having to unwrap the `Bot`s manually.

Do you have any example of where we would want this instance?